rust/phoenix-sdk-core/src/packet_decoder.rs

```diff
@@ -0,0 +1,192 @@
+use borsh::{BorshDeserialize, BorshSerialize};
+use phoenix::state::{SelfTradeBehavior, Side};
+
+#[derive(BorshDeserialize, BorshSerialize)]
+pub enum OrderPacketEnum {
+    PostOnly,
+    Limit,
+}
+
+#[derive(BorshDeserialize, BorshSerialize)]
+pub struct PostOnlyPacketDeprecated {
```

Comment:
prefer `DeprecatedPostOnlyPacket`, and also for current version to live on top of deprecated version, and also an aggressive doc saying that this shouldn't be used

Reply:
I can probably make this private. It's only used as helper for deserialization